### PR TITLE
Forms Automated Test - LPS-132278 | master

### PR DESCRIPTION
**Pull request by piresdm**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase [hunk 1]
```diff
@@ -253,8 +253,40 @@ definition {
...
+			fieldPositionNumber = "0",
+			fieldType = "Multiple Selection");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 1",
+			optionValue = "1");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 2",
+			optionValue = "2");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase:274]: Hey @piresdm, can we rename this macro to `addOption` only?

Apparently, it was created firstly to attend Single Selection field needs, after all its current name is `addSingleSelectionOption`, although this same macro is being invoked on tests related to the fields **Single Selection**, **Multiple Selection** and **Select From List**. Therefore, in order to better reflect the actual use of this macro would be important we rename it. :slightly_smiling_face:

File: modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro [hunk 2]
```diff
@@ -1093,6 +1093,12 @@ definition {
 		}
 	}
 
+	macro viewPredefinedValueDropdownContent {
+		AssertElementPresent(
+			key_text = "${key_textContent}",
```
Comment (natocesarrego) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro:1098]: @piresdm, please rename `key_text` to `key_textContent` to keep naming consistent. :wink:

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 3]
```diff
@@ -66,6 +66,16 @@ definition {
 		Click(locator1 = "FormFields#OPTIONS_DELETE_ROW_BUTTON");
 	}
 
+	macro deleteSpecificOptionFromMultipleSelection {
+		Click(
+			key_value = "${key_valueOption}",
+			locator1 = "FormFields#DELETE_OPTION_FROM_MULTIPLE_SELECTION");
+
+		AssertElementNotPresent(
+			key_value = "${key_valueOption}",
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:75]: @piresdm, please rename `key_option` to `key_valueOption` to keep naming consistent. :wink:

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 4]
```diff
@@ -669,6 +669,12 @@
 	<td>//td[contains(@class,'lfr-image-column')]</td>
 	<td></td>
 </tr>
+<!--MULTIPLE_SELECTION-->
+<tr>
+	<td>DELETE_OPTION_FROM_MULTIPLE_SELECTION</td>
+	<td>//input[contains(@value,'${key_value}')]//..//..//..//..//button[contains(@class,'close')]</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:675]: @piresdm, can we try to create a path more specific?

I'm concerned about the maintenance of this path in the long term due to UI changes.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase (hunk 1 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase:655]: @natocesarrego like we talked before, this macro was used in many other PR`s that were already merged. So, I did not change it...

File: modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro (hunk 2 above)
Comment (piresdm) [reply modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro]: @natocesarrego done!

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro (hunk 3 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: @natocesarrego done!

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path (hunk 4 above)
Comment (piresdm) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path]: @natocesarrego done!

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**